packages/node_modules/@webex/plugin-meetings/sample/index.html

```diff
@@ -116,14 +228,15 @@ <h2>Then click:</h2>
       </fieldset>
     </form>
     <div>
-      <video style="max-height: 100px, max-width: 100px" id="remotevideo" autoplay></video>
-      <video style="max-height: 100px, max-width: 100px" id="screenvideo" autoplay></video>
-      <video style="max-height: 100px, max-width: 100px" id="localvideo" muted="true" autoplay></video>
-      <video style="max-height: 100px, max-width: 100px" id="localscreen" autoplay></video>
+      <video style="--width: 20%;width: var(--width);height: calc(var(--width) * 0.75);" id="localvideo" muted="true" autoplay></video>
+      <video style="--width: 35%;width: var(--width);height: calc(var(--width) * 0.75);" id="remotevideo" autoplay></video>
+      <video style="--width: 35%;width: var(--width);height: calc(var(--width) * 0.75);" id="sharevideo" autoplay></video>
+      <video style="--width: 20%;width: var(--width);height: calc(var(--width) * 0.75);" id="localshare" autoplay></video>
```

Comment:
could probably have moved these styles up to the style block you added but it's fine